### PR TITLE
submodule nits

### PR DESCRIPTION
**Pull request by rwalker-apple**
#### Problem
 irregular configs for submodules

 #### Summary of Changes
 make regular

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -41,7 +41,7 @@
 	ignore = dirty
 	commit = 35be9760b4edf485287f78c0cf652beb19c047f0
 [submodule "pigweed"]
-	path = third_party/pigweed
+	path = third_party/pigweed/repo
```
Comment (gerickson) [inline .gitmodules:44]: Thank you!

**Review by gerickson — APPROVED** [review]

**Review by woody-apple — APPROVED** [review]

**Comment by rwalker-apple** [comment]
I don't know how to fix the lgtm failure off hand.  I think I could have a PR where pigweed is mapped in twice?  hope we can land without lgtm

**Comment by mspang** [comment]
> I don't know how to fix the lgtm failure off hand. I think I could have a PR where pigweed is mapped in twice? hope we can land without lgtm

Try:

git mv third_party/pigweed pigweed                 
mkdir -p third_party/pigweed
git mv pigweed third_party/pigweed/repo

**Review by saurabhst — APPROVED** [review]

**Review by BroderickCarlin — APPROVED** [review]

**Comment by rwalker-apple** [comment]
@woody-apple this is ready

**Review by turon — APPROVED** [review]
Looks good.  Thanks.

**Comment by woody-apple** [comment]
@jelderton @hawk248 @andy31415 ?

**Comment by woody-apple** [comment]
@rwalker-apple can we fix the conflicts here?

**Comment by rwalker-apple** [comment]
@woody-apple ready

**Review by andy31415 — APPROVED** [review]

**Pull request merged**